### ISSUE TITLE
Feat: "Clean" install during setup.

### ISSUE THREAD

**Issue by nynymike** [JanssenProject/jans#2053]
To make Jans easy to learn, we add a bunch of sample clients, scripts,  and other configuration (for example a sample custom attribute). We should have a setup option for `clean install`, where we only install the minimal configuration to make Jans function.

**Comment by ossdhaval** [comment]
@nynymike yes, this is needed. My suggestion on implementation would be to have clean install by default, and have a switch or flag in case if dev-tooling is required. Something similar to `--dev`. When this switch is used during installation, it'll install sample clients, scripts etc plus it should also load test data. The ability to load test data is already available using `--args="-t"`.
